lib/public/js/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 console.log('Hello, pals!');
+
+const setActiveNavItems = () => {
+
+    const list = document.getElementsByClassName('nav__item');
+    const length = list.length;
+    const currentPage = window.location.href;
+
+    let iterator;
+    for (iterator = 0; iterator < length; ++iterator) {
+        const item = list[iterator];
+        if (item.href === currentPage) {
+            item.classList.add('nav__item--active');
```

Comment:
We can do this in handlebars too if that's preferable.

Reply:
ahh... never thought of that. was feeling too hapi